x/earn/testutil/suite.go

```diff
@@ -205,7 +204,7 @@ func (suite *Suite) GetEvents() sdk.Events {
 // AddCoinsToModule adds coins to the earn module account
 func (suite *Suite) AddCoinsToModule(amount sdk.Coins) {
 	// Does not use suite.BankKeeper.MintCoins as module account would not have permission to mint
-	err := simapp.FundModuleAccount(suite.BankKeeper, suite.Ctx, types.ModuleName, amount)
+	err := suite.App.FundModuleAccount(suite.Ctx, types.ModuleName, amount)
```

Comment:
lots of warm and fuzzies, deleting code is nice but whats great is when you can simplify an interface, 👨🏽‍🍳 💋